### PR TITLE
feat(Icons): MoneyBackGuarantee

### PR DESCRIPTION
**Pull request by LinKCoding**
### Overview

This adds the MoneyBackGuarantee Icon used currently in Brand. 
Also removes the fill and normalizes it to "none" like with the other icons.
Subsequent PR to remove the icon from Brand.

### PR Checklist

- [x] Related to JIRA ticket: [GM-895](https://skillsoftdev.atlassian.net/browse/GM-895)
- [x] I have run this code to verify it works
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

1. Go to ?path=/story/atoms-icons-regular--editable-icons&args=iconName:MoneyBackGuaranteeIcon
2. At the top, select Canvas
3. In the `iconName` field, go to Control row and select `MoneyBackGuaranteeIcon` 
4. Check that it loads properly on both light and dark modes
5. Go check monolith's preview + mono's preview
7. ...
8. Profit!

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/38755)  | [Monolith Env](https://pr-38755-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/8354)  | [Portal Env](https://3603755d.brand-storybook.pages.dev/)   

[GM-895]: https://skillsoftdev.atlassian.net/browse/GM-895?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.4-alpha.c77664.0<br/>@codecademy/gamut-icons@9.32.1-alpha.c77664.0<br/>@codecademy/gamut-kit@0.6.452-alpha.c77664.0<br/>@codecademy/styleguide@68.2.4-alpha.c77664.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://67293b29ed0488743dbb6986--gamut-preview.netlify.app>https://67293b29ed0488743dbb6986--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/67293b29ed0488743dbb6986>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.4-alpha.cf2a08.0<br/>@codecademy/gamut-icons@9.32.1-alpha.cf2a08.0<br/>@codecademy/gamut-kit@0.6.452-alpha.cf2a08.0<br/>@codecademy/styleguide@68.2.4-alpha.cf2a08.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.4-alpha.bcd7d9.0<br/>@codecademy/gamut-icons@9.32.1-alpha.bcd7d9.0<br/>@codecademy/gamut-kit@0.6.452-alpha.bcd7d9.0<br/>@codecademy/styleguide@68.2.4-alpha.bcd7d9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://672a4def000bd4ef017987b3--gamut-preview.netlify.app>https://672a4def000bd4ef017987b3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/672a4def000bd4ef017987b3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.4-alpha.01a171.0<br/>@codecademy/gamut-icons@9.32.1-alpha.01a171.0<br/>@codecademy/gamut-kit@0.6.452-alpha.01a171.0<br/>@codecademy/styleguide@68.2.4-alpha.01a171.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://672b84835901b31ec6670a8d--gamut-preview.netlify.app>https://672b84835901b31ec6670a8d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/672b84835901b31ec6670a8d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.4-alpha.07f8f5.0<br/>@codecademy/gamut-icons@9.32.1-alpha.07f8f5.0<br/>@codecademy/gamut-kit@0.6.452-alpha.07f8f5.0<br/>@codecademy/styleguide@68.2.4-alpha.07f8f5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://672bb0d6fae399123fb3d53b--gamut-preview.netlify.app>https://672bb0d6fae399123fb3d53b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/672bb0d6fae399123fb3d53b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.5-alpha.4d030a.0<br/>@codecademy/gamut-icons@9.32.2-alpha.4d030a.0<br/>@codecademy/gamut-kit@0.6.453-alpha.4d030a.0<br/>@codecademy/styleguide@68.2.5-alpha.4d030a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://672bd946f7316d516cf2f6b4--gamut-preview.netlify.app>https://672bd946f7316d516cf2f6b4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/672bd946f7316d516cf2f6b4>Deploy Logs</a></p>

File: packages/gamut/src/Video/__tests__/Video.test.tsx [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
 
 const renderView = setupRtl(Video, {});
 
 describe('Video', () => {
-  it('loads a video with a vimeo URL', () => {
-    const { view } = renderView({
-      videoUrl: 'https://vimeo.com/145702525',
-      videoTitle: 'Super Science Friends',
-    });
+  it('loads a video with a vimeo URL', async () => {
```
Comment (jakemhiller) [inline packages/gamut/src/Video/__tests__/Video.test.tsx:9]: are these changes still needed?

Comment (LinKCoding) [reply packages/gamut/src/Video/__tests__/Video.test.tsx:9]: I believe the tests were passing but had some warnings related to this. 

@dreamwasp, what do you think?

Comment (jakemhiller) [reply packages/gamut/src/Video/__tests__/Video.test.tsx:9]: i'd recommend trying to fix this without wrapping the whole thing in `act`. Often converting `getBy*` to `await findBy*` is enough to get rid of those warnings.  `view.findBy*` is also a simpler version of `await waitFor(() => view.getBy*)`

```diff
--- view.getByTitle('Super Science Friends');
+++ await view.findByTitle('Super Science Friends');
```

Comment (dreamwasp) [reply packages/gamut/src/Video/__tests__/Video.test.tsx:9]: we can remove from this PR, jake's other stuff prob fixed the need for it

Comment (dreamwasp) [reply packages/gamut/src/Video/__tests__/Video.test.tsx:9]: i'll remove!

Comment (LinKCoding) [reply packages/gamut/src/Video/__tests__/Video.test.tsx:9]: ah, gotcha, okies, it's easier if I just remove it then :) thanks Jake and Cass!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.5-alpha.d3adaa.0<br/>@codecademy/gamut-icons@9.32.2-alpha.d3adaa.0<br/>@codecademy/gamut-kit@0.6.453-alpha.d3adaa.0<br/>@codecademy/styleguide@68.2.5-alpha.d3adaa.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://672ce313949be5cc1039594d--gamut-preview.netlify.app>https://672ce313949be5cc1039594d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/672ce313949be5cc1039594d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.5-alpha.d3adaa.0<br/>@codecademy/gamut-icons@9.32.2-alpha.d3adaa.0<br/>@codecademy/gamut-kit@0.6.453-alpha.d3adaa.0<br/>@codecademy/styleguide@68.2.5-alpha.d3adaa.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.4.5-alpha.d3adaa.0<br/>@codecademy/gamut-icons@9.32.2-alpha.d3adaa.0<br/>@codecademy/gamut-kit@0.6.453-alpha.d3adaa.0<br/>@codecademy/styleguide@68.2.5-alpha.d3adaa.0<br/></pre>
